File: C/PRAK203-2210817310001-MuhammadRyanRizkyRahmadi.c
#include <stdio.h>

int main (){
    float a, b, i, j, x, y;
    
    printf ("Nilai A =");
    scanf("%f", &a);
    printf ("Nilai B =");
    scanf("%f", &b);
     printf ("Nilai I =");
    scanf("%f", &i);
     printf ("Nilai J =");
    scanf("%f", &j);
     printf ("Nilai X =");
    scanf("%f", &x);
     printf ("Nilai Y =");
    scanf("%f", &y);
        float nilaixx = (a-b)*(i/j)-(x+y);
    printf("Hasil dari A kurang B dikali dengan I di agi J dikurang dengan X ditambah Y adalah %.3f", nilaixx);

}

File: Python/PRAK203-2210817310001-MuhammadRyanRizkyRahmadi.py
import math
nilaiA= float(input("Nilai A="))
print, nilaiA
nilaiB= float(input("Nilai B="))
print, nilaiB
nilaiI= float(input("Nilai I="))
print, nilaiI
nilaiJ= float(input("Nilai J="))
print, nilaiJ
nilaiX= float(input("Nilai X="))
print, nilaiX
nilaiY= float(input("Nilai Y="))
print, nilaiY

hasil = (nilaiA-nilaiB)*(nilaiI/nilaiJ)-(nilaiX+nilaiY)
print("Hasil dari A kurang B dikali dengan I di agi J dikurang dengan X ditambah Y adalah {:.3f}".format(hasil))